tests/signing_integ_test.rs

```diff
@@ -26,14 +26,18 @@ fn test_two_party_sign() {
         &party_one_first_message,
         &party_two_first_message.d_log_proof,
     );
-    assert!(party_one_second_message.d_log_proof_result.is_ok());
+    party_one_second_message
+        .d_log_proof_result
+        .expect("Party one DLog proved.");
 
     let party_two_proof_result = party_two::KeyGenSecondMsg::verify_commitments_and_dlog_proof(
         &ec_context,
         &party_one_first_message,
         &party_one_second_message,
     );
-    assert!(party_two_proof_result.d_log_proof_result.is_ok());
+    party_two_proof_result
+        .d_log_proof_result
+        .expect("Party two DLog proved.");
```

Comment:
Since the `expect(...)` message is used to [describe the error context](file:///home/roman/.rustup/toolchains/stable-x86_64-unknown-linux-gnu/share/doc/rust/html/std/result/enum.Result.html#examples-17), consider rephrasing it to a "negative" form:
```
"Party # DLog proved" -> "Incorrect party # DLog proof"
```

Reply:
Interesting, It sound counter intuitive to me. I read it "expect party dlog proved" instaed of "expect Incorrect party..". Nevermind, you are correct, I will change that.